qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -78,6 +76,7 @@
             inputs = input1
             input_dev_bus_type_input1 = virtio
             input_dev_type_input1 = mouse
+            vmcoreinfo = yes
```

Comment:
Hi leidong, I'm just curious that this patch is for removing qemufwcfg, so why add vmcoreinfo cfg here?

Reply:
Ok, get it. vm will not add this device without this cfg.